### PR TITLE
New pkl files in alpha and updated alphamodel.py

### PR DESCRIPTION
**Pull request by ChuyangXiao**
Good afternoon, Professor! First, I am so sorry for the occasional errors that occurred during the modeling process. After a careful examination, it is due to our misunderstanding with some of the data and thus we updated our pred_prep_dic dataset in the folder alpha, and use the function get_embedding from the ANN instead of creating a new PKL file. Do you mind checking this out? In addition, thank you so much for solving the issue of reading PKL this morning!

### CODE REVIEW

File: services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py [hunk 1]
```diff
@@ -1,43 +1,35 @@
...
+
+            with open('/usr/src/app/src/alpha/gbdt_model_v3.pickle', 'rb') as f:
+                GBDT_model = pickle.load(f)
+            with open('/usr/src/app/src/alpha/content_artist_style_dic.pickle', 'rb') as f:
+                dic_id_style = pickle.load(f)
+            with open('/usr/src/app/src/alpha/prediction_prep_dic.pickle', 'rb') as f:
+                prep_dic = pickle.load(f)
+        #    with open('/usr/src/app/src/alpha/dic_id_to_embedding.pickle', 'rb') as f:
+        #        dic_id_embed = pickle.load(f)
+
```
Comment (kennethgoodman) [inline services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py:32]: are you sure this is what you want? I think my code will be much much faster as it isn't loading these models each time and only installing the package once

File: services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py [hunk 2]
```diff
@@ -89,10 +82,17 @@ def predict_probabilities(self, content_ids, user_id, seed=None, **kwargs):
                 )
             )
             
+<<<<<<< HEAD
```
Comment (kennethgoodman) [inline services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py:85]: you need to remove these

Comment (ChuyangXiao) [reply services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py]: Thank you for your recommendation. Now I have removed the commented lines and used the index / content_id dictionary to get the id_to_embedding directly from the PKL file. Do you mind taking a look at the new ones?

**Review by kennethgoodman — APPROVED** [review]

**Comment by kennethgoodman** [comment]
columbia-e4579-api-1  |   File "/usr/src/app/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py", line 90
columbia-e4579-api-1  |     except:
columbia-e4579-api-1  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
columbia-e4579-api-1  | SyntaxError: default 'except:' must be last

This broke production, reverting

**Pull request merged**